tests/integration/helpers/cluster.py

```diff
@@ -2814,6 +2846,26 @@ def wait_ldap_to_start(self, timeout=180):
 
         raise Exception("Can't wait LDAP to start")
 
+    def wait_local_kms_to_start(self, timeout=15):
```

Comment:
IMO it'd make sense to increase the timeout to say 2-3 mins

Reply:
Roger. I'll increase it. In my tests it took only a few ms to spawn the Local KMS service, but better safe than sorry